src/layouts/mobile-layout.tsx

```
@@ -1,18 +1,64 @@
-        {showFooter && <Footer />}
+        {showFooter && (
+          <div className="fixed bottom-0 left-1/2 -translate-x-1/2 w-full max-w-[400px] text-[10px] font-medium z-10 text-center">
+            <div className="relative">
+              <img src={Background} className="w-full" alt="푸터 배경" />
+
+              <div className="absolute bottom-0 w-full aspect-[400/92] flex justify-between px-6">
+                <div className="flex justify-center w-19 pt-2">
+                  <button
+                    className="flex flex-col cursor-pointer gap-2"
+                    onClick={() => navigate("/")}
+                  >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)